debezium-core/src/main/java/io/debezium/relational/history/KafkaDatabaseHistory.java

```diff
@@ -51,15 +35,15 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import io.debezium.DebeziumException;
-import io.debezium.annotation.NotThreadSafe;
-import io.debezium.config.Configuration;
-import io.debezium.config.Field;
-import io.debezium.config.Field.Validator;
-import io.debezium.document.DocumentReader;
-import io.debezium.relational.HistorizedRelationalDatabaseConnectorConfig;
-import io.debezium.util.Collect;
-import io.debezium.util.Threads;
+import java.io.IOException;
+import java.time.Duration;
+import java.time.temporal.ChronoUnit;
+import java.util.*;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
+import java.util.function.Consumer;
```

Comment:
Did the `impsort` plugin rearrange these imports?  I doubt that it did so lets reorganize these back to the order they had previously.